### ISSUE TITLE
panic

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/eris-pm#150]
```
Error compiling contracts
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x555e60]

goroutine 1 [running]:
panic(0xa86ba0, 0xc8200120b0)
	/usr/lib/go/src/runtime/panic.go:481 +0x3e6
github.com/eris-ltd/eris-pm/perform.DeployJob(0xc82021a790, 0xc8200a5680, 0x0, 0x0, 0x7f9953806460, 0xc820264db0)
	/go/src/github.com/eris-ltd/eris-pm/perform/jobs_contracts.go:68 +0xdf0
github.com/eris-ltd/eris-pm/perform.RunJobs(0xc8200a5680, 0x0, 0x0)
	/go/src/github.com/eris-ltd/eris-pm/perform/job_manager.go:87 +0xdc9
github.com/eris-ltd/eris-pm/packages.RunPackage(0xc8200a5680, 0x0, 0x0)
	/go/src/github.com/eris-ltd/eris-pm/packages/run_package.go:19 +0xb2
github.com/eris-ltd/eris-pm/cmd.RunPackage(0x10a7840, 0xc820141400, 0x0, 0x14)
	/go/src/github.com/eris-ltd/eris-pm/cmd/epm.go:109 +0x23
github.com/eris-ltd/eris-pm/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).execute(0x10a7840, 0xc820088010, 0x14, 0x15, 0x0, 0x0)
	/go/src/github.com/eris-ltd/eris-pm/Godeps/_workspace/src/github.com/spf13/cobra/command.go:477 +0x3f4
github.com/eris-ltd/eris-pm/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0x10a7840, 0x0, 0x0)
	/go/src/github.com/eris-ltd/eris-pm/Godeps/_workspace/src/github.com/spf13/cobra/command.go:551 +0x46a
github.com/eris-ltd/eris-pm/cmd.Execute()
	/go/src/github.com/eris-ltd/eris-pm/cmd/epm.go:79 +0x62
main.main()
	/go/src/github.com/eris-ltd/eris-pm/cmd/epm/main.go:8 +0x14
```

**Comment by VoR0220** [comment]
This means that it cant call the compilers server. What address are you calling at?

**Comment by compleatang** [comment]
that shouldn't panic.... even if it's the wrong address.

**Comment by VoR0220** [comment]
It shouldn't you are correct. Should do a better job error handling.

**Comment by VoR0220** [comment]
@zramsay can you try reproducing this error now?

**Comment by VoR0220** [comment]
fixed this with #151
